frontend/src/constants.ts

```diff
@@ -1,6 +1,7 @@
 export const APP_NAME = process.env.VUE_APP_NAME || "";
 export const APP_URL = process.env.VUE_APP_URL || "";
 export const LOGIN_URL = process.env.VUE_APP_LOGIN_URL || "";
+export const CLIENT_ID = process.env.VUE_APP_CLIENT_ID || "";
```

Comment:
We also need to add this variable to `.env.template`.

Reply:
done